lib/receptor_controller/client/response_worker.rb

```diff
@@ -59,6 +70,28 @@ def stop
       end
     end
 
+    # Listening for the consumer call to the Fetch API
+    # Then releasing original starting thread
+    def init_notifications(init_lock, init_wait)
+      ActiveSupport::Notifications.subscribe('request.connection.kafka') do |*args|
```

Comment:
I'd like to see a log message here, that way we know it got that far that it's waiting for the client to initialize, maybe something like:

`Initializing Kafka Client...`
then
`Kafka Client successfully initialized.`

---

The current log message doesn't quite makes sense since it isn't a receptor response.